src/Pos/Communication/Limits.hs

```diff
@@ -283,6 +284,12 @@ instance MessageLimited (MsgHeaders ssc) where
         return $
             MsgHeaders <$> vectorOf Const.recoveryHeadersMessage headerLimit
 
+-- TODO: Update once we move to CBOR.
+instance MessageLimitedPure MsgSubscribe where
+    msgLenLimit = 0
```

Comment:
1. TODO here looks obsolete.
2. Limit = 0 is suspicious, is it really 0? I saw that we encode it as 42 somewhere above.